tests/test_integration.py

```diff
@@ -1251,6 +1251,80 @@ def test_global_modeling_with_events_and_future_regressors():
             fig3 = m.plot_parameters()
 
 
+def test_trend_global_local_modeling():
```

Comment:
Let's separate these tests and the previous global modeling tests into a separate test_global.py file